Mobile/ShopLens/Droid/Source/AppPreferences.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using Android.Content;
+
+namespace ShopLens.Droid.Source
+{
+    public class ActivityPreferences
+    {
+        ISharedPreferences prefs;
+        ISharedPreferencesEditor prefsEditor;
+        Context pContext;
+        List<string> items;
```

Comment:
`items` might be just a local variable, it's not used anywhere except `GetPreferencesToList` I think

Reply:
Fixed.